### PR TITLE
change comment flags to magic flags

### PR DESCRIPTION
**Pull request by pete88b**
hope this one is not too painful to review (o:

### CODE REVIEW

File: nbdev/export.py [hunk 1]
```diff
@@ -219,7 +223,7 @@ def _f(gps):
 
 # Cell
 def extra_add(code):
-    "Catch adds to `__all__` required by a cell with `_all_=`"
+    "Catch adds to `__all__` required by a cell with `%nbdev_add2all`"
```
Comment (sgugger) [inline nbdev/export.py:226]: Mention both:
"Catch adds to `__all__` required by a cell with `_all_=` or `%nbdev_add2all`."

File: nbs/00_export.ipynb [hunk 2]
```diff
@@ -4,36 +4,37 @@
    "cell_type": "code",
```
Comment (sgugger) [inline nbs/00_export.ipynb:4]: <p>There is some weird MarkDown in this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

File: nbdev/export.py (hunk 1 above)
Comment (pete88b) [reply nbdev/export.py]: done

File: nbs/00_export.ipynb (hunk 2 above)
Comment (pete88b) [reply nbs/00_export.ipynb:4]: I think I've upset the ReviewNB parser/renderer again )o:

here's how it looks on github:
image

here's how it looks when I build docs locally:
image

Comment (pete88b) [reply nbs/00_export.ipynb:4]: In case you're interested, I've created https://github.com/ReviewNB/support/issues/63 to see if they can fix this

Comment (sgugger) [reply nbs/00_export.ipynb:4]: Thanks for reporting the issue!

**Comment by sgugger** [comment]
Thanks for all your work on this!

**Pull request merged**